### ISSUE TITLE
Patched

### ISSUE THREAD

**Issue by Tripp-dripp** [imsmartbutdumb/iready-pro-hacks#1]
It's broken, Iready auto closes it now.

**Comment by 1Glacier1** [comment]
Will it get fixed?

**Comment by n1wu** [comment]
> e

i dont think it will be fixed anytime soon but you could use the inspect element skip/pass lesson

also now my grade is going to go down because i use the hack to bring up my grades (6th grade)

**Comment by 1Glacier1** [comment]
Hello I was one of the people using this hack it also bring my grade up thank you so much (7th grade). I found another hack that works decent it’s the same pop up (I ready overload). I will try to link it or make a video on how to use it when I get home.

**Comment by 1Glacier1** [comment]
https://github.com/cupiditys/iReady-Overload

**Comment by ClippedOPP** [comment]
None of the things work. not even the lesson skipper which I need very badly. If anyone has a lesson skipper that I can use I would really appreciate it.
